docs/user-guide/creating-application/deployment-template.md

```diff
@@ -1,7 +1,39 @@
 # Devtron Charts
 
-This documentation consists of the charts available on Devtron.
+A deployment configuration is a manifest for the application. It defines the runtime behavior of the application.
+
+Devtron includes both predefined deployment template charts and custom charts created by a super admin.
```

Comment:
Devtron includes deployment template for both default as well as custom charts created by a super admin.